### DISCUSSION TITLE
It is actually possible to animate the Collapse <detail><summary>. See example. :)

### DISCUSSION THREAD

**Discussion by sjardim** [saadeghi/daisyui#2594] [Ideas / feature requests]
The detail/summary tags are animated using a 'margin hack'. 🤓 

Here's an example on Tailwind Play: https://play.tailwindcss.com/BGWglTvMfW

Do you think we can use something like this?

---
I implemented this today after seeing an answer on Stack Overflow: https://stackoverflow.com/a/73447722 by Tarcio Zemel - https://github.com/tarciozemel

**Comment by NikarashiHatsu** [comment]
Interesting...
